lib/rdoc/generator/darkfish.rb

```diff
@@ -797,14 +800,22 @@ def excerpt(comment)
 
     # Match from a capital letter to the first period, discarding any links, so
     # that we don't end up matching badges in the README
-    first_paragraph_match = text.match(/[A-Z][^\.:\/]+\./)
-    return text[0...150].gsub(/\n/, " ").squeeze(" ") unless first_paragraph_match
+    pattern = ParagraphExcerptRegexpUnicode
+    begin
+      first_paragraph_match = text.match(pattern)
+    rescue Encoding::CompatibilityError
+      # The doc is non-ASCII text and encoded in other than Unicode base encodings.
+      raise unless pattern.eaual?(ParagraphExcerptRegexpUnicode)
```

Comment:
Should this be `equal?`?

Reply:
Fixed at eb3a5ba6f2e38199758266124847e57174314179